projects/DiffusionDet/README.md

```diff
@@ -0,0 +1,115 @@
+## Description
+
+This is an implementation of [DiffusionDet](https://github.com/ShoufaChen/DiffusionDet) based on [MMDetection](https://github.com/open-mmlab/mmdetection/tree/3.x), [MMCV](https://github.com/open-mmlab/mmcv), and [MMEngine](https://github.com/open-mmlab/mmengine).
+
+<center>
+<img src="https://user-images.githubusercontent.com/48282753/211472911-c84d658a-952b-4608-8b91-9ac932cbf2e2.png">
+</center>
+
+## Usage
+
+<!-- For a typical model, this section should contain the commands for training and testing. You are also suggested to dump your environment specification to env.yml by `conda env export > env.yml`. -->
+
+### Training commands
+
+In MMDetection's root directory, run the following command to train the model:
+
+```bash
+python tools/train.py projects/DiffusionDet/configs/xxx.py
+```
+
+For multi-gpu training, run:
+
+```bash
+python -m torch.distributed.launch --nnodes=1 --node_rank=0 --nproc_per_node=${NUM_GPUS} --master_port=29506 --master_addr="127.0.0.1" tools/train.py projects/DiffusionDet/configs/xxx.py
+```
+
+### Testing commands
+
+In MMDetection's root directory, run the following command to test the model:
+
+```bash
+python tools/test.py projects/DiffusionDet/configs/xxx.py ${CHECKPOINT_PATH}
+```
+
+## Results
+
+Here we provide the baseline version of DiffusionDet with ResNet50 backbone.
+
+To find more variants, please visit the [official model zoo](https://github.com/ShoufaChen/DiffusionDet#models).
+
+| Backbone |  Style  | Lr schd | Mem (GB) | FPS | mask AP val2017 |    Config    |         Download         |
+| :------: | :-----: | :-----: | :------: | :-: | :-------------: | :----------: | :----------------------: |
+|   R-50   | PyTorch |    x    |    x     |  x  |        x        | [config](<>) | [model](<>) \| [log](<>) |
+
+## License
+
+TODO: DiffusionDet is using CC-BY-NC 4.0 license.
+
+## Citation
+
+If you find DiffusionDet is useful in your research or applications, please consider giving a star 🌟 to the [official repository](https://github.com/ShoufaChen/DiffusionDet) and citing DiffusionDet by the following BibTeX entry.
+
+```BibTeX
+@article{chen2022diffusiondet,
+      title={DiffusionDet: Diffusion Model for Object Detection},
+      author={Chen, Shoufa and Sun, Peize and Song, Yibing and Luo, Ping},
+      journal={arXiv preprint arXiv:2211.09788},
+      year={2022}
+}
+```
+
+## Checklist
```

Comment:
Can the following be deleted?

Reply:
Seems This need keep to tell users, which step this project is done